src/screens/LandingScreen/SpeakerSection.js

```diff
@@ -138,9 +138,9 @@ const SpeakerSection = () => {
                     pb: [1, 3]
                   }}
                 >
-                  Gaurav is a Software Engineer and a Content Creator. He has
-                  YouTube channel on system design and algorithms has more than
-                  300k subscribers.
+                  Gaurav is a Software Engineer and a Content Creator. His
+                  YouTube channel on system design and algorithms boasts more
```

Comment:
boasts? Are you sure about this sentence?

Reply:
@HarishTeens It was Vedant's suggestion :P

Reply:
lol okay